core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -10,7 +10,7 @@
   </div>
 </div>
 <div class="modal-body">
-  <div ng-if="!textToTranslate">
+  <div ng-if="!textToTranslate && !loadingData">
```

Comment:
While the content is loading we are expecting to see the `<lazy-loading>` directive, is that still the same? Can you please share a video of the UI on the slow network?

Reply:
Good point. Looks like the `<lazy-loading>` directive was nested under the `textToTranslate` check. I moved the `<lazy-loading>` directive out of that check.

Video: https://drive.google.com/file/d/1Egs16-89Zjpoqdyjh5HgYcdMuoEU50WC/view?usp=sharing